python/src/otel/otel_sdk/otel_wrapper.py

```diff
@@ -0,0 +1,82 @@
+import os
+import logging
+from opentelemetry import trace
+from importlib import import_module
+
+from opentelemetry.sdk.extension.aws.trace import AwsXRayIdsGenerator
+
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    SimpleExportSpanProcessor,
+    BatchExportSpanProcessor,
+)
+
+from opentelemetry.resource import AwsLambdaResourceDetector
+
+from opentelemetry.instrumentation.aws_lambda import AwsLambdaInstrumentor
+from pkg_resources import iter_entry_points
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# TODO: get_aggregated_resources
+resource = Resource.create().merge(AwsLambdaResourceDetector().detect())
+trace.set_tracer_provider(
+    TracerProvider(
+        ids_generator=AwsXRayIdsGenerator(),
+        resource=resource,
+    )
+)
+
+console_exporter = os.environ.get("OTEL_DEBUG", None)
+if not console_exporter is None:
+    from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+
+    trace.get_tracer_provider().add_span_processor(
+        SimpleExportSpanProcessor(ConsoleSpanExporter())
+    )
+    logger.info("Console exporter initialized.")
+
+ci = os.environ.get("_OTEL_CI", None)
```

Comment:
CI shouldn't leak into the SDK, this should just be using the `OTEL_TRACES_EXPORTER` variable from the spec. Though it makes me wonder what this file is for? Doesn't python support the variables natively? We use that support for the java lambda wrappers so no special config wrapper

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestWrapperBase.java#L23

Reply:
good.